intermediate-python-debugging/exercise-using-debugger.md

```diff
@@ -0,0 +1,76 @@
+# Problem Set: Using the Debugger
+
+In this exercise we will use the debugger to identify two errors.  To run the activity clone the code at [AdaGold/python-debugging](https://github.com/AdaGold/python-debugging).  Then set up the project by creating a virtual environment and installing pytest.
+
+The code contains two bugs, one runtime error crashes the program on execution and the other is a logic error which calculates the incorrect sum for an Order instance.  
+
+## Activity
```

Comment:
Is this a problem set or activity (check top header and second header and file name)?

In the current curriculum:

Problem Sets are pieces of (ideally) one-night homework. The file should be prefixed `problem-set` (not exercise). Problem sets are something we want to ask students to definitely complete, so usually there's some Learn challenge autograded component to it.

Activities are designed to replace a roundtable, so are ideally written to be one-hour small group activities.

In either case, at some point, it'd be great if it followed a [curriculum template](https://github.com/Ada-Developers-Academy/core/wiki/Templates-and-Stubs_). I recognize that it kinda sucks because you prob didn't know about this when you wrote it, so if you don't convert it to this template, I will eventually :D !

Reply:
One more note, in my ideal world, the header and file name should match exactly, including the "the"

`activity-using-the-debugger` for "Activity: Using the Debugger"
or `activity-using-debugger` for "Activity: Using Debugger"